lib/candy_check/play_store.rb

```diff
@@ -1,15 +1,16 @@
-require "google/apis/androidpublisher_v3"
+require "google-apis-androidpublisher_v3"
+require "googleauth"
 
-require "candy_check/play_store/android_publisher_service"
```

Comment:
For code formatting in this project we use [Rufo](https://github.com/ruby-formatter/rufo)

Please try to avoid adding "diff noise" (like quotes to single-quotes below) by running Rufo before commiting your changes.